src/main/java/com/coolkids/coolKidsApp/services/EventService.java

```diff
@@ -9,9 +9,12 @@ public interface EventService {
 
     List<EventDTO> getAllEvents();
 
-    EventDTO getEventById(String id);
```

Comment:
We have a merge request. It might be because you didnt pull the newly merged changes. Can you resolve it and see what changes you want to keep?